app/stylesheets/CameraNew.scss

```diff
@@ -0,0 +1,9 @@
+@import "./support/colors";
+.tilt-icon.tilt-icon-camera-new {
+  height: 18px;
+  width: 23px;
+  fill: white;
```

Comment:
do you need a white fill if the only thing drawn is the `icon-group` which gets overridden?